test/gui/preset_settings/test_starting_area.py

```diff
@@ -107,3 +106,66 @@ def test_check_credits(skip_qtbot, preset_manager):
 
     checkbox_list = window._starting_location_for_node
     assert checkbox_list.get(not_expected, None) == None
+
+@pytest.fixture
+def setup_and_teardown_for_multiple_nodes(preset_manager):
+    # TODO: Currently all areas have only one sp that's why the test is artificially constructed.
```

Comment:
Add one such case to the Blank game and use that for the test.

Reply:
Rewrote the test. Added a node to blank game. Also added test/checks for correct behaviour of world box (therefore marked another node in another area as start point).
After the push it just came to my mind that it maybe was not necessary to add a node and instead use an already existing one.